eng/build.yml

```diff
@@ -0,0 +1,79 @@
+#Your build pipeline references a secret variable named ‘PB_PublishBlobFeedKey’. Create or edit the build pipeline for this YAML file, define the variable on the Variables tab, and then select the option to make it secret. See https://go.microsoft.com/fwlink/?linkid=865972
+#Your build pipeline references a secret variable named ‘PB_PublishBlobFeedUrl’. Create or edit the build pipeline for this YAML file, define the variable on the Variables tab, and then select the option to make it secret. See https://go.microsoft.com/fwlink/?linkid=865972
+#Your build pipeline references the ‘_BuildConfig’ variable, which you’ve selected to be settable at queue time. Create or edit the build pipeline for this YAML file, define the variable on the Variables tab, and then select the option to make it settable at queue time. See https://go.microsoft.com/fwlink/?linkid=865971
+#Your build pipeline references the ‘BuildPlatform’ variable, which you’ve selected to be settable at queue time. Create or edit the build pipeline for this YAML file, define the variable on the Variables tab, and then select the option to make it settable at queue time. See https://go.microsoft.com/fwlink/?linkid=865971
+
+parameters:
+  agentOs: $(agentOs)
+  queue: $(queue)
+
+phases:
+  - phase: ${{ parameters.agentOs }}
+    queue: ${{ parameters.queue }}
+    steps:
+    - ${{ if notIn(variables['Build.Reason'], 'IndividualCI', 'BatchedCI', 'PullRequest') }}:   
+      - task: ms-vseng.MicroBuildTasks.30666190-6959-11e5-9f96-f56098202fef.MicroBuildSigningPlugin@1
+        displayName: Install Signing Plugin
+        inputs:
+          signType: '$(PB_SignType)'
+        condition: and(succeeded(), in(variables['PB_SignType'], 'real', 'test'))
+
+    - ${{ if eq(parameters.agentOs, 'Windows_NT') }}:
+      - script: build\CIBuild.cmd 
+                  -configuration %BuildConfig%
+                  /p:PB_PublishBlobFeedKey=%BlobFeedKey%
+                  /p:PB_PublishBlobFeedUrl=%BlobFeedUrl%
+        displayName: Build
+        env:
+          BuildConfig: $(_BuildConfig)
+          BlobFeedKey: $(PB_PublishBlobFeedKey)
+          BlobFeedUrl: $(PB_PublishBlobFeedUrl)
+
+    - ${{ if eq(parameters.agentOs, 'Linux') }}:
+      - script: build/cibuild.sh 
+                  --configuration $BuildConfig
+                  /p:PB_PublishBlobFeedKey=$BlobFeedKey
+                  /p:PB_PublishBlobFeedUrl=$BlobFeedUrl
+        displayName: Build
+        env:
+          BuildConfig: $(_BuildConfig)
+          BlobFeedKey: $(PB_PublishBlobFeedKey)
+          BlobFeedUrl: $(PB_PublishBlobFeedUrl)
+
+    - ${{ if notIn(variables['Build.Reason'], 'IndividualCI', 'BatchedCI', 'PullRequest') }}:   
```

Comment:
I don't think some of the below steps will actually work as-is.  Again, I presume you're ok with that and intend to fix them up when you do official build changes.  Depending on your timeline, you may just want to delete the internal build only sections for the sake of simplicity.  ie, remove the install MicroBuild plugin task, and the artifact publishing stuff.

Reply:
If you do remove the internal build stuff, then you might just simplify even further and collapse this whole thing into a single file (ie, move the "cibuild" stuff into the `.vsts-ci.yml` file.

Reply:
I will keep them. I will work on getting official builds working today, as soon as this PR makes in.